getting_started/step_by_step/scripting.rst

```diff
@@ -331,7 +331,7 @@ Also, remember that nodes are referenced by name, not by type.
 
 .. note::
 
-    The right-hand panel of the connect dialogue is for binding specific
+    The 'advanced' panel of the connect dialogue is for binding specific
```

Comment:
The panel in question only appears once the 'Advanced' slider is toggled. It seems more intuitive to refer to it like this instead.